lib/rules/no-restricted-browser-globals-during-ssr.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+'use strict';
+const { noReferenceDuringSSR } = require('../rule-helpers');
+const { docUrl } = require('../util/doc-url');
+const { browser } = require('globals');
+
+const forbiddenGlobalNames = new Set(Object.keys(browser));
+const allowedGlobals = [
```

Comment:
I am suspecting that you came to this list by looking at the globals exposed in Node. 

If it's the case my recommendation is to diff the `browsers` with the `node` properties exposed by `globals`. This way we won't have to maintain this list moving forward.